src/org/jitsi/impl/neomedia/rtcp/RTCPTCCPacket.java

```diff
@@ -539,21 +518,22 @@ public RTCPTCCPacket(long senderSSRC, long sourceSSRC,
             else if (seqDelta % 7 == 3)
```

Comment:
if there's some stuff we could copy-paste from the spec, i think that'd be nice.  i know it's not you, just the standard, but seeing code like `if (seqDelta % 7 == 3)` just feels so arbitrary and weird to read